### DISCUSSION TITLE
AmazonSmile

### DISCUSSION THREAD

**Discussion by ralphieq** [moralmunky/Home-Assistant-Mail-And-Packages#399] [Q&A] [answered]
Hi,

My wife has recently started using AmazonSmile when placing her Amazon orders. AmazonSmile donates a small portion of your purchase to charitable organizations. As a result those packages stopped getting tracked by the addon. The only difference in the Amazon.com emails I see is in the subject line and the Amazon logo imbedded within the email. Everything else is formatted the same as a normal Amazon purchase and tracking email.

AmazonSmile email Subject line for example:
Your AmazonSmile order #xxx-xxxxxxx-xxxxxxx has shipped
VS
Your Amazon.com order #xxx-xxxxxxx-xxxxxxx has shipped

From email address doesn't change. It is still `Amazon.com <shipment-tracking@amazon.com>
`
Not sure if Amazon has this charity feature outside the US.  How would I add AmazonSmile to the regex when parsing the subject to include them in the Amazon package tracking?

Any help is sincerely appreciated!

**Comment by firstof9** [comment, marked as answer]
There is no subject regex for the "shipped" Amazon emails, the integration should pick them up.

**Reply by ralphieq** [reply]
First off, thank you for the quick response! Appreciate all the work that's gone into the addon. 

My wife has an Amazon order she made through Amazon Smile coming tomorrow. I'll see if the sensor picks it up and report back.

![image](https://user-images.githubusercontent.com/5899401/110247077-51129300-7f38-11eb-82c6-f78d132382d1.png)
